### PR TITLE
Recipe for steeltoe boots

### LINKED ISSUE #12770: Steeltoed boots cannot be cut up
Experimental v. c8854c6
Steeltoed boots cannot be cut up for materials. I get a message saying they are made of a material that cannot be cut up. They are part Steel, so I guess that part kind of make sense. I would think they should give at least a piece or two of leather and maybe some scrap metal.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/22922506-steeltoed-boots-cannot-be-cut-up?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by EricFedrowisch**
Added recipe for steeltoe boots so that they can be disassembled. Closes #12770

### CODE REVIEW

**Comment by Coolthulhu** [comment]
It would be better as a deconstruction-only recipe.

Otherwise it needs to be higher level and require more advanced tools. Something like lvl 4, hammering 2 or 3 and welder or forge.

**Comment by ArikVhedrovix** [comment]
I would be glad to make it a deconstruction only recipe as that is my intent. I thought that with no skill book recipes and autolearn set to false, it would only be usable when cutting up found boots. Did I not understand correctly?

**Comment by Coolthulhu** [comment]
Check out recipes_deconstruction.json. Recipes there all have the same "template".
Also, as a deconstruction-only recipe, it should also be declared in recipes_deconstruction.json.

**Comment by ArikVhedrovix** [comment]
Ok. Does this look better?

**Comment by Coolthulhu** [comment]
Bump the time, but the metal sawing requirement can be removed.

Also, check out what is the buildbot complaining about.

Other than that it looks good

File: data/json/recipes/recipe_deconstruction.json [hunk 1]
```diff
@@ -804,6 +804,24 @@
     ]
   ]
 },{
+    "type" : "recipe",
+    "result": "boots_hiking",
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_deconstruction.json]: `"boots_hiking"`?

**Comment by ArikVhedrovix** [comment]
Bump time up or down? Buildbot caught an extra comma at end of "components". Fixed that and removing the metal saw need.

**Comment by Coolthulhu** [comment]
Up. Though maybe it's not required. 800 may be fine.

Also, check the item id. You've got `boots_hiking`, not `boots_steel`.

**Comment by ArikVhedrovix** [comment]
Argh! Lol. Thanks for all the help. I will fix name and move to 1000 time.

**Comment by ArikVhedrovix** [comment]
@Coolthulhu  Thanks again!

**Pull request merged**